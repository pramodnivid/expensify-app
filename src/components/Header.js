import React from "react"
import ReactDOM from "react-dom"
import {BrowserRouter,Route,Switch,NavLink} from "react-router-dom"

const Header = ()=>(
    <div>
        <h1>Expensify</h1>

        <NavLink to="/" activeClassName="is-active" exact={true}> Home </NavLink>
        <br/>

        <NavLink to="/create" activeClassName="is-active">Add Expense</NavLink>
        <br/>

        <NavLink to="/help" activeClassName="is-active">Help Expense</NavLink>

</div>
)
export default Header;