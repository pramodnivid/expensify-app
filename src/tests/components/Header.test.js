import React from "react"
import ReactShallowRenderer from "react-test-renderer/shallow"
import Header from "../../components/Header"
import {shallow} from "enzyme";
//import toJSON from "enzyme-to-json"
test('should render header correctly',()=>{
    const wrapper = shallow(<Header />);
    //expect(wrapper.find("h1").text()).toBe("Expensify")
    expect((wrapper)).toMatchSnapshot()
    // const renderer = new ReactShallowRenderer();

    // renderer.render(<Header />)

    // expect(renderer.getRenderOutput()).toMatchSnapshot();

    // console.log(renderer.getRenderOutput())
});