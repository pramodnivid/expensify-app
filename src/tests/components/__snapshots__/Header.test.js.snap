// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<div>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
     Home 
  </NavLink>
  <br />
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/create"
  >
    Add Expense
  </NavLink>
  <br />
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/help"
  >
    Help Expense
  </NavLink>
</div>
`;
